crates/networking/rpc/net/mod.rs

```diff
@@ -0,0 +1,15 @@
+use serde_json::Value;
+
+use crate::{
+    utils::{RpcErr, RpcRequest},
+    RpcApiContext,
+};
+
+pub fn version(_req: &RpcRequest, context: RpcApiContext) -> Result<Value, RpcErr> {
+    let chain_spec = context
+        .storage
+        .get_chain_config()
+        .map_err(|error| RpcErr::Internal(error.to_string()))?;
```

Comment:
Do we need this map_err? Can't we just use the question mark like we do in `engine/fork_choice.rs:106`? We are calling `get_chain_config` there as well. The store error should correctly propagate as a `RpcErr`.

This is because the `From<StoreError> for RpcErr` trait is implemented in `rpc/utils.rs`

Reply:
ah i didn't know that. Will remove the explicit map_err()!